src/main/java/appeng/tile/networking/TileCableBus.java

```diff
@@ -233,7 +233,8 @@ public void onReady()
 	@Override
 	public boolean requiresTESR()
 	{
-		return this.getCableBus().isRequiresDynamicRender();
+		return true;
+		// return this.getCableBus().isRequiresDynamicRender();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3ACommentedOutCodeLine)